lib/internal/errors.js

```diff
@@ -136,6 +136,14 @@ E('ERR_ENCODING_INVALID_ENCODED_DATA',
   (enc) => `The encoded data was not valid for encoding ${enc}`);
 E('ERR_ENCODING_NOT_SUPPORTED',
   (enc) => `The "${enc}" encoding is not supported`);
+E('ERR_EVENTS_MAX_LISTENERS_TYPE',
+  '"defaultMaxListeners" must be a positive number');
+E('ERR_EVENTS_UNHANDLED_ERROR',
+  (err) => {
+    const msg = 'Unhandled "error" event.';
+    if (!err) return msg;
```

Comment:
Using `if (err !== undefined)` would be better as there might be other falsy values out there that someone throws.